### ISSUE TITLE
I need this with caja

### ISSUE THREAD

**Issue by trappedinspacetime** [Stunkymonkey/nautilus-open-any-terminal#40]
Can it be easily ported to work with caja?

**Comment by Stunkymonkey** [comment]
have you tried `sudo apt-get install caja-open-terminal`? remember to run `caja -q` to test it.

**Comment by trappedinspacetime** [comment]
@Stunkymonkey Thank you for responding. I already had that but I needed to launch another terminal with `sixel` support. 
I attempted to add a right click context menu via `caja-action-configuration` tool but I failed to launch it in the current directory, instead it would open it in the parent directory.
Today I managed to fix it via a trick:

       bash -c "gnome-terminal --working-directory=$( echo "$1" | cut -b 8- )"

I congrat you on your project.
Regards.

**Comment by Stunkymonkey** [comment]
`caja` is indeed a fork of `nautilus`.

maybe you can look at the source code of those projects and integrate them somehow. so I you manage to support `caja` as well, I am open for contributions.

**Comment by trappedinspacetime** [comment]
@Stunkymonkey I'm sorry I am late at responding to you. I agree with you about it should be possible to port it to `Caja`. Today I tried to port it to work with `Caja` but I failed. Caja also provide `python3-caja python-caja-common` but they may differ in paths and class names. I noticed it when I needed to port `gedit plugins` to `pluma plugins`.
I am alien to `Python`. I need to debug and test it but i's beyond my coverage for the time being. I am sorry since I can not be of help. 
`caja-action-configuration` method works well.
I 'll inform you when I have any progress.
Regards.
